app/services/datastream_importer.rb

```diff
@@ -16,6 +16,15 @@ def initialize(datastream_filename)
   def import!
     Xccdf::Benchmark.transaction do
       save_all_benchmark_info
+      import_remediations
     end
   end
+
+  private
+
+  def import_remediations
+    RemediationsAPI.new(
+      Account.find_by!(account_number: ENV['JOBS_ACCOUNT_NUMBER'])
```

Comment:
I was going to comment on this - but you got it covered on https://github.com/RedHatInsights/e2e-deploy/commit/464d17cd4edd8be2711d8578a41179d8710977b0